src/msgraph_core/base_graph_request_adapter.py

```diff
@@ -1,12 +1,12 @@
 import httpx
-from kiota_abstractions.authentication import AuthenticationProvider
+from kiota_abstractions.authentication import AuthenticationProvider  # type: ignore
```

Comment:
We do not have type annotations for our kiota-libraries, if we did, we would have to install them alongside these lbraries so that mypy is satisfied for type checking, for now the option we have is to silence mypy n these, that is what #type:ignore does